nb-engine/nb-engine-cli/src/main/java/io/nosqlbench/engine/cli/atfiles/NBAtFile.java

```diff
@@ -167,8 +168,9 @@ private static LinkedList<String> interposePath(LinkedList<String> formatted, Pa
         while (iter.hasNext()) {
             String word = iter.next();
             String modified = word.replaceAll("\\$\\{DIR}",parent.toString());
+            Optional<String> interpolatedString = NBEnvironment.INSTANCE.interpolate(modified);
```

Comment:
Simply adding orThrow() here would address my earlier concerns. If there is an ENV reference which fails to lookup, we should treat it as a hard error.

Reply:
Working on it.